src/platform/tls_schannel.c

```diff
@@ -746,6 +746,11 @@ CxPlatTlsSetClientCertPolicy(
             "SetCredentialsAttributesW(SECPKG_ATTR_CLIENT_CERT_POLICY) failed");
     }
 
+    if (SecStatus == SEC_E_UNSUPPORTED_FUNCTION)
```

Comment:
Not so about this, `SecStatusToQuicStatus` does not actually map anything, it just casts, so this would end up as an unknown status at the app layer.